### PR TITLE
regroup tag added, more verbose error messages

### PR DESCRIPTION
**Pull request by mikes222**

### CODE REVIEW

File: lib/src/buildin_tags/regroup.dart [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  }
+
+  @override
+  bool get hasEndTag => false;
+
+  @override
+  bool get defaultAsProcessing => false;
+}
+
+extension JaggedIterable<T> on Iterable<Iterable<T>> {
```
Comment (stefanrusek) [inline lib/src/buildin_tags/regroup.dart:85]: Prefix name with an underscore

File: lib/src/exception/parseblock_exception.dart [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import '../model.dart';
+
+class ParseblockException implements Exception {
```
Comment (stefanrusek) [inline lib/src/exception/parseblock_exception.dart]: Capital b

File: lib/src/exception/tagrender_exception.dart [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import '../tag.dart';
+
+class TagrenderException implements Exception {
```
Comment (stefanrusek) [inline lib/src/exception/tagrender_exception.dart]: Capital r

**Review by stefanrusek — CHANGES_REQUESTED** [review]
Just a couple naming issues

**Comment by mikes222** [comment]
Updated the master branch to reflect your suggestions.

**Comment by mikes222** [comment]
Do I need to do some additional things for the updated pull request?

**Comment by stefanrusek** [comment]
Fix failed check. Try 'dart format -l 200' to fix the formatting

**Comment by mikes222** [comment]
done. expressions.dart has been changed (3 blank lines inserted)

**Pull request merged**